include/darknet.h

```diff
@@ -1032,6 +1032,10 @@ LIB_API void diounms_sort(detection *dets, int total, int classes, float thresh,
 // network.h
 LIB_API float *network_predict(network net, float *input);
 LIB_API float *network_predict_ptr(network *net, float *input);
+#ifdef CUDA_OPENGL_INTEGRATION
```

Comment:
I think I'm going to remove this `ifdef`, it doesn't seem harmful to leave the function definition in, and it's causing some issues when I try to generating Rust bindings to this code.